### ISSUE TITLE
Forget to fix monet icon?

### ISSUE THREAD

**Issue by 0xrxL** [AbdurazaaqMohammed/AntiSplit-M#114]
Shouldn't it have been fixed already? APK word is still not visible. 🤔

![Screenshot_20241028_231528_One UI Home](https://github.com/user-attachments/assets/663fd1e2-2439-4050-b1b0-6c40d2eef859)

**Comment by federicogith** [comment]
@AbdurazaaqMohammed

**Comment by AbdurazaaqMohammed** [comment]
I changed the icon of the app again. I think this icon is more in line with a monet theme

![Image](https://github.com/user-attachments/assets/3d34356a-b3f1-4b92-910b-de431481559b)

![Image](https://github.com/user-attachments/assets/f0da7558-f1f4-4894-a12e-0680214ee6b6)

**Comment by 0xrxL** [comment]
> I changed the icon of the app again. I think this icon is more in line with a monet theme
> 
> ![Image](https://github.com/user-attachments/assets/3d34356a-b3f1-4b92-910b-de431481559b)
> 
> ![Image](https://github.com/user-attachments/assets/f0da7558-f1f4-4894-a12e-0680214ee6b6)

Yeah...at this point was inevitable. The icon, which I previously posted, was structurally glitchy with monet.

Thanks however.
